configure.ac

```diff
@@ -6802,6 +6852,7 @@ AC_PLUGIN([dns],                 [$with_libpcap],             [DNS traffic analy
 AC_PLUGIN([dpdkevents],          [$plugin_dpdkevents],        [Events from DPDK])
 AC_PLUGIN([dpdkstat],            [$plugin_dpdkstat],          [Stats from DPDK])
 AC_PLUGIN([drbd],                [$plugin_drbd],              [DRBD statistics])
+AC_PLUGIN([dpdk_telemetry],      [$plugin_dpdk_telemetry],    [Metrics from DPDK Telemetry])
```

Comment:
Please, keep list sorted.

Reply:
Addressed and available in latest commit.

Reply:
@rpv-tomsk , Can you please approve the pull request , if no  more comments.

Reply:
Sorry, I'm `untrusted contributor` and have no such permission.